src/servers/tests/http/authorize.rs

```diff
@@ -65,6 +66,12 @@ async fn test_http_auth() {
     );
 }
 
+#[tokio::test]
+async fn test_http_auth() {
+    check_http_auth(http::header::AUTHORIZATION.as_str()).await;
+    check_http_auth(AUTHORIZATION_HEADER).await;
+}
+
 #[tokio::test]
 async fn test_schema_validating() {
```

Comment:
nitpick: test both case for `test_schema_validating`

Reply:
fixed in https://github.com/GreptimeTeam/greptimedb/pull/5389/commits/d80e0c031d49610c0282f6c7370cd5b461b237f9

Reply:
Do you want to fix this comment? @v0y4g3r